src/java.base/share/classes/java/lang/classfile/Signature.java

```diff
@@ -158,6 +161,7 @@ public static ClassTypeSig of(ClassDesc className, TypeArg... typeArgs) {
          */
         public static ClassTypeSig of(ClassTypeSig outerType, ClassDesc className, TypeArg... typeArgs) {
```

Comment:
Another instance of nullable argument... I thought @asotona expressed that we wish to convert nullable args to `Optional`. Need to make such parameters consistent across the ClassFile API.